frontend/src/components/Edu.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div>
+    <input
+      type="text"
+      v-model="education.degree"
+      class="degree"
+      placeholder="Degree"
+      @input="saveEducation"
+    />
+
+    <input
+      type="text"
+      v-model="education.university"
+      placeholder="University"
+      @input="saveEducation"
+    />
+  </div>
+
+  <div>
+    <label>From</label>
+    <input
+      type="date"
+      v-model="education.startingYear"
+      @input="saveEducation"
+    />
+
+    <label>></label>
+    <input
+      type="date"
+      v-model="education.graduationYear"
+      @input="saveEducation"
+    />
+  </div>
+
+  <div>
```

Comment:
<img width="458" alt="Screenshot 2024-01-23 at 12 07 56" src="https://github.com/philippOkie/resume-builder-vue/assets/48657633/c53d4505-78f6-41c5-a2f4-965a0ea4cae0">

Reply:
might need to double check the filename. **Readability counts**